prerequisites.rst

```diff
@@ -1,35 +1,250 @@
+======  =================  ==============  ==============  =======================================
+ OS      Debian Linux       oldoldstable    stable          Current stable (plus 1 month released)
+ BOOT    BIOS,UBOOT,UEFI    BIOS,mbr        BIOS,gpt        Disable Secure boot, gpt table
+ SDS     HDD,SSD,USB...     2 partitions    3 partitions    Root with 120G size, 8G swap size
+ DDS     HDD,SSD,USB...     1 partition     1 per share     One disk or part per shared resource
+ NET     LAN,WAN,SAN,VPN    LAN             SAN,PAN,LAN     Fiber ipv4, or at least cable LAN
+======  =================  ==============  ==============  =======================================
+
+The operating system (OS)
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The |omv| is a piece of software, it resides and runs under a key software OS
+named Debian GNU/Linux as "the universal operating system" [9]_ 
+
+Supported Debian OS versions can be checked in the :doc:`releases section </releases>`
+
+While possible to deploy in a virtual environment, depending of the nature of
+virtualization it will degrade the performance access in various ways. Installation
+in LXC or any other container based solution is not supported.
+
+Whatever the situation, the |omv| assumes that target operating system does not
+have any previous installation of any of the programs used by |omv| dependencies:
+
+=============  ==============  =================  ==========================================
+ software       package          related to        Observations
+=============  ==============  =================  ==========================================
+ http server    nginx           |webui|            port 80 must be free at installation
+ http system    php             |webui|            only OS php packages wil be used!
+ display GUI    lighdm,xdm..    Desktop install    **Conflicts, any will be deinstalled**
+ network man    netplan.io      Networking         network is managed by |omv| with netplan
+ ssh server     ssh             Remote access      any configuration will be managed
+ mbs server     samba           Shared drives      any configuration will be managed
+ quota man      quota           Quota manage       any configuration will be managed
+=============  ==============  =================  ==========================================
+
+The device boot (BOOT)
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+|omv| will install on any kind of boot device and this will relies on the
+operating system that is Debian. The downloable ISO image ready to use from
+the oficial web page only supports at the moment amd64 with BIOS mode boot.
+
+The System drive storage (SDS)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The system storage disk will be fully managed and taken over by the |omv| after
+installed. Any brand is supported as is also supported by the Debian system.
+For Flash Drive type storage devices as well as solid state drives being used as
```

Comment:
Spaghetti sentence. Please rephrase.

Reply:
Spaghetti sentence fixed with commit c441ec7d9649da3fb60e9f23635c041b62f2a54c, also complete rewrite the paragraph